prdoc/pr_4522.prdoc

```diff
@@ -0,0 +1,34 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Added runtime support for reporting BEEFY fork voting
+
+doc:
+  - audience:
+    - Runtime Dev
+    - Runtime User
+    description: |
+      This PR adds the `report_fork_voting`, `report_future_voting` extrinsics to `pallet-beefy`
```

Comment:
I would maybe mention that you need to specify `AncestryHelper` for `pallet_beefy::Config` - it is breaking change, right?

Reply:
Yes. Good point. Will mention it.

Reply:
Done